src/user-icon/user-icon.scss

```diff
@@ -136,60 +136,91 @@
       color: t(iui-text-color-muted);
     }
   }
-}
-
-@mixin iui-user-icon-list {
-  display: flex;
 
-  .iui-user-icon:not(:first-child) {
-    margin-left: -$iui-xs;
+  &.iui-user-icon-count:focus {
+    @include themed {
+      box-shadow: t(iui-focus-box-shadow);}
+    border-radius: 50%;
 
-    .iui-initials {
+    &.iui-user-icon > .iui-stroke {
       @include themed {
-        box-shadow: 0 0 0 $iui-xs t(iui-color-background-1);
-      }
+        box-shadow: t(iui-focus-box-shadow);}
     }
+  }
+}
 
-    &.iui-small {
-      margin-left: -$iui-xxs;
+@mixin iui-user-icon-list {
+  display: inline-flex;
+  flex-wrap: wrap;
 
-      .iui-initials {
-        @include themed {
-          box-shadow: 0 0 0 $iui-xxs t(iui-color-background-1);
-        }
-      }
+  .iui-user-icon:not(:first-child) {
+    margin-left: $iui-xs;
+   
+    &.iui-small {
+      margin-left: $iui-xxs;
     }
+ 
 
     &.iui-large {
-      margin-left: -$iui-s;
+      margin-left: $iui-s;
     }
 
     &.iui-x-large {
-      margin-left: -$iui-sm;
+      margin-left: $iui-sm;
     }
   }
```

Comment:
Not sure these values are supposed to be the same as iui-animated. (I would expect more space between the smaller icons).

But if these values are correct, then maybe extra the rules into a mixin that can be used here and in iui-animated.

Reply:
The values should match animated ones once they are fully expanded. Should I create that additional rule or is it good enough for now?

Reply:
Yeah, I'd say create a mixin for reusability

Reply:
added @mixin iui-user-icon-margins